CONTRIBUTING.md

```diff
@@ -0,0 +1,19 @@
+# Contributing
+
+Thank you for investing your time in contributing to our project!
+
+Read our [Code of Conduct](https://github.com/vektra/mockery/blob/master/CODE_OF_CONDUCT.md) to keep our community approachable and respectable.
+
+## Local development setup
+
+This project uses [Taskfile](https://taskfile.dev/), please see the [Install Taskfile](https://taskfile.dev/installation/) and
+[Taskfile usage documentation](https://taskfile.dev/usage/) for more information.
+
+Once installed, run `task -l` for list of valid targets.
```

Comment:
You shouldn't have to reference the installation docs for taskfile, we actually import all tools mockery uses in the `tools/` sub-module so you can just do `go mod download -x` to get everything.